cmd/configtxgen/main.go

```diff
@@ -32,6 +32,13 @@ import (
 var logger = flogging.MustGetLogger("common.tools.configtxgen")
 
 func doOutputBlock(config *genesisconfig.Profile, channelID string, outputBlock string) error {
+	if config.Orderer != nil && config.Orderer.OrdererType == "BFT" {
```

Comment:
I don't think this is the right place to put it. 

It should be deeper into the stack. 

This is the `main.go`.  It should not contain validation logic. 

There is validation logic in `common/channelconfig/orderer.go`, maybe we can add it there?

```

func (oc *OrdererConfig) Validate() error {
	for _, validator := range []func() error{
		oc.validateBatchSize,
		oc.validateBatchTimeout,
	} {
		if err := validator(); err != nil {
			return err
		}
	}

	return nil
}
```